app/experimenter/kinto/serializers.py

```diff
@@ -0,0 +1,75 @@
+from time import time
+from rest_framework import serializers
+
+from experimenter.experiments.models import Experiment, ExperimentVariant
+
+
+class ExperimentRapidBranchesSeraizlier(serializers.ModelSerializer):
+    value = serializers.SerializerMethodField()
+
+    class Meta:
+        model = ExperimentVariant
+        fields = ("slug", "ratio", "value")
+
+    def get_value(self, obj):
+        # placeholder value
+        return {}
+
+
+class ExperimentRapidArgumentSerializer(serializers.ModelSerializer):
+    slug = serializers.ReadOnlyField(source="normandy_slug")
+    userFacingName = serializers.ReadOnlyField(source="name")
+    userFacingDescription = serializers.ReadOnlyField(source="public_description")
+    active = serializers.ReadOnlyField(default=True)
+    isEnrollmentPaused = serializers.ReadOnlyField(default=False)
+    proposedEnrollment = serializers.ReadOnlyField(source="proposed_enrollment")
+    bucketConfig = serializers.SerializerMethodField()
+    startDate = serializers.SerializerMethodField()
+    endDate = serializers.ReadOnlyField(default=None)
+    branches = ExperimentRapidBranchesSeraizlier(many=True, source="variants")
+    referenceBranch = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Experiment
+        fields = (
+            "slug",
+            "userFacingName",
+            "userFacingDescription",
+            "active",
+            "isEnrollmentPaused",
+            "features",
+            "proposedEnrollment",
+            "bucketConfig",
+            "startDate",
+            "endDate",
+            "branches",
+            "referenceBranch",
+        )
+
+    def get_bucketConfig(self, obj):
+        return {
+            "randomizationUnit": "normandy_id",
+            "namespace": "",
+            "start": 0,
+            "count": 0,
+            "total": 10000,
+        }
+
+    def get_referenceBranch(self, obj):
+        control_branch = obj.variants.get(is_control=True)
+        return control_branch.slug
+
+    def get_startDate(self, obj):
+        # placeholder value
+        return int(time())
```

Comment:
Oh yeah that's a good point we didn't specify which date/time format to use, or did we....

Oh hrmmm.... the TS schema says `Date` which I think is a js `Date` object, which isn't correct.  I know @mythmon prefers to use ISO format instead of unix timesteamps, so we should probably use that:

https://docs.python.org/3/library/datetime.html#datetime.datetime.isoformat

Reply:
And we probably have to change the schema in the shared repo to say string:

https://github.com/mozilla/nimbus-shared/blob/main/types/experiments.ts#L38

@k88hudson ?

Reply:
Latest version of the schema now has `date-time` for `format` https://137-267090370-gh.circle-artifacts.com/0/artifacts/nimbus-shared-0.0.3/schemas/experiments/ExperimentRecipe.json